code/modules/mob/new_player/new_player.dm

```diff
@@ -261,6 +261,7 @@
 	if(!is_job_whitelisted(src, rank))	 return 0
 	if(!job.player_old_enough(client))	return 0
 	if(job.admin_only && !(check_rights(R_EVENT, 0))) return 0
+	if(job.mentor_only && !(check_rights(R_ADMIN|R_MENTOR, FALSE))) return FALSE
```

Comment:
Styling here is conflicting with the rest of the codebase. The checks above do as well though

Reply:
Yeah I'd rather not make minor styling fixes here that upstream hasn't fixed, it'll just create useless conflicts down the line.